test/utilities.py

```diff
@@ -13,6 +13,10 @@ def chdir(dirname=None):
     curdir = os.getcwd()
     try:
         if dirname is not None:
+            if dirname == 'files':
+                import test
+                dirname = os.path.split(test.__file__)[0]
+                dirname = os.path.join(dirname, 'files')
```

Comment:
Generalized this a bit to allow the directory to be found outside the test directory.